pkgs/applications/video/mpv/scripts/sponsorblock-minimal.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildLua
+, fetchFromGitea
+, curl
+}:
+
+# Usage: `pkgs.mpv.override { scripts = [ pkgs.mpvScripts.sponsorblock-minimal ]; }`
```

Comment:
This should already be documented elsewhere, please remove

Reply:
Done.